app/repository.py

```diff
@@ -0,0 +1,72 @@
+from sqlalchemy.exc import SQLAlchemyError
+from app.database import Base, engine, SessionLocal
+from dataclasses import dataclass
+
+
+# creates database
+Base.metadata.create_all(engine)
+
+
+@dataclass
+class Err:
+    error: str
+
+
+NO_OBJECT_ERROR = "No such object!"
```

Comment:
Do we really need it as a constant?

Reply:
maybe not, it's just cause it's used few times in the file, so to avoid possible spelling mistakes etc. You advise not to have it as constant in such case?